ci/ci.sh

```diff
@@ -743,10 +743,13 @@ build() {
 }
 
 run_minimal_test() {
+  EXPECTED_PYTHON_VERSION=$1
   BAZEL_EXPORT_OPTIONS="$(./ci/run/bazel_export_options)"
   # Ignoring shellcheck is necessary because if ${BAZEL_EXPORT_OPTIONS} is wrapped by the double quotation,
   # bazel test cannot recognize the option.
   # shellcheck disable=SC2086
+  bazel test --test_output=streamed --config=ci --test_env=RAY_MINIMAL=1 --test_env=EXPECTED_PYTHON_VERSION=$EXPECTED_PYTHON_VERSION ${BAZEL_EXPORT_OPTIONS} python/ray/tests/test_minimal_install
```

Comment:
I was thinking of an "easier" approach where we just asserted the python version in shell here like check the `python --version` output. 

What's the motif for running another pytest and assert in there?

Reply:
I'm not sure how bazel works, so wanted to verify that bazel picks up right version. If someone confident in bazel behavior says we can trust it, I can delete the test.

Reply:
I don't have a strong opinion here - I' m ok with the current approach.